app/src/main/java/ch/epfl/sweng/swenggolf/database/CreateUserActivity.java

```diff
@@ -21,10 +22,13 @@
 
 public class CreateUserActivity extends AppCompatActivity {
 
-    private EditText mail;
     private EditText name;
     private ImageView photo;
     private User user;
+    private static final String PRESENTATION_TEXT = "Welcome"
```

Comment:
It is better to put this string in res/string, so that we have all strings at the same place and that will ease the process of translating, for instance. (Additionally there is a typo, you forgot the 'd' of 'and')

Reply:
Ok thanks, I'll do that 👍